Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'puma', '~> 5.0'
 gem 'jsonapi.rb', '~> 1.7'
 # Use Active Storage variant
 # gem 'image_processing', '~> 1.2'
-
+gem 'magic_frozen_string_literal', '~> 1.2'
```

Comment:
Had no idea there was a gem for this, hopefully it won't become [the next left-pad-gate](https://medium.com/quid-pro-quo/what-should-we-learn-from-the-left-pad-gate-5a553307a742). ;)